events/event-manager.js

```diff
@@ -0,0 +1,55 @@
+var eventModels = require('./event-models.js');
+var Event = eventModels.Event;
+// This function return an object containing the event UID of the newly created event.
+var createEvent = function(reqData, locals, createCallback) {
```

Comment:
We've recently decided to use ES6 syntax for anonymous functions like these, so this line should now look like ```var createEvent = (reqData, locals, createCallback) => {```